metadata-ingestion/setup.py

```diff
@@ -121,7 +121,14 @@ def get_long_description():
     "greenlet",
 }
 
-sqllineage_lib = "sqllineage==1.3.6"
+sqllineage_lib = {
+    "sqllineage==1.3.6",
+    # We don't have a direct dependency on sqlparse but it is a dependency of sqllineage.
+    # As per https://github.com/reata/sqllineage/issues/361
+    # and https://github.com/reata/sqllineage/pull/360,
+    # sqllineage has compat issues with sqlparse 0.4.3 and 0.4.4.
```

Comment:
Think it's just 0.4.3 (they're saying to pin it to 0.4.3 on https://github.com/reata/sqllineage/pull/360)